CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```

Comment:
Should we just point people to [the main OpenCue CONTRIBUTING file](https://github.com/imageworks/OpenCue/blob/master/CONTRIBUTING.md) here? I think we'll want to follow the same guidelines, right?

Reply:
Good point. Actually, there doesn't seem to be much consistency between the other 'Docsy' sites. I'll update the file to ask contributors to review the main repo's guidelines. Eventually, we'll need to add some docs-specific guidelines on the opencue.io site, such as style guide...etc, but we could do that in a page on the site.